projects/aca-about/tsconfig.lib.prod.json

```diff
@@ -2,7 +2,12 @@
 {
   "extends": "./tsconfig.lib.json",
   "compilerOptions": {
-    "declarationMap": false
+    "declarationMap": false,
+    "paths": {
```

Comment:
does about component need these mappings?

Reply:
Yes, in order to build this library, we need to have already built dependant libraries (we cannot build librarie when dependant library e.g. `aca-shared` is located outside the `aca-about` library root folder)